_episodes/03-types-conversion.md

```diff
@@ -260,21 +260,33 @@ first is 2 and second is 5
 > 4. A lab specimen's age.
 > 5. Current population of a city.
 > 6. Average population of a city over time.
+>
+> > ## Solution
+> >
+> > 1.  _integer_. Number of days since the start of the year.
```

Comment:
I can think of cases in which these answers are too proscriptive, eg, what plotting, if data is taken at: 
8am on the 3rd, 
noon on the 4th, 
and 6pm on the 4th, 
I can imagine using x-values of 3.333, 4.5, 4.75  

Similarly, a city's population of 141,283 is approximate because of births, dealths, and migration.  Saying the city has a population of 1.41E5 people (using float) doesn't seem wrong to me.

In general, do we need answers for every exercise?  I'm inclined to say no, given te variety of answers some of these questions have.